src/redist/targets/GenerateLayout.targets

```diff
@@ -5,7 +5,6 @@
     <_PublishSingleFileForRid>true</_PublishSingleFileForRid>
     <_PublishSingleFileForRid Condition="'$(RID)' == 'win-x86'">false</_PublishSingleFileForRid>
     <_GenerateLayoutDependsOnTargets>CleanLayoutPath;PublishDotnetCoreUninstallProject</_GenerateLayoutDependsOnTargets>
-    <_GenerateLayoutDependsOnTargets Condition="'$(RID)' == 'win-x86'">CleanLayoutPath;PublishDotnetCoreUninstallProject;SignLayout</_GenerateLayoutDependsOnTargets>
```

Comment:
`SignLayout` is also now handled by Arcade now

Reply:
Why did we lose the condition?

Reply:
Because it is identical to the previous line, but adds the `SignLayout` target (which is now unused)

Reply:
Good point